task/mypy-lint/0.1/README.md

```diff
@@ -0,0 +1,84 @@
+# Mypy Linter for Python
+
+`Mypy` is an optional `static type checker` for `Python` that aims to combine the benefits of dynamic (or "duck") typing and static typing. `Mypy` combines the expressive power and convenience of Python with a powerful type system and compile-time type checking. `Mypy` type checks standard Python programs; run them using any Python VM with basically no runtime overhead.
+
+The following task is used to provide static analysis on python files mounted using `mypy` (Python linter).
+
+## Installing the Task
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/task/mypy-lint/0.1/mypy-lint.yaml
+```
+
+## Parameters
+
+- **args**: The extra params along with the file path needs to be provided as the part of `args`. (_Default_: `["--help"]`)
+
+## Workspaces
+
+- **shared-workspace** : The workspace containing files on which we want to apply linter check. It can be a shared workspace with the `git-clone` task or a `ConfigMap` mounted containing some files.
```

Comment:
```suggestion
- **shared-workspace**: The workspace containing files on which we want to apply linter check. It can be a shared workspace with the `git-clone` task or a `ConfigMap` mounted containing some files.
```